tests/resources/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Match Aurora MySQL version as closely as possible
+FROM mysql:5.7.33
```

Comment:
Please downgrade MySql version to 5.7.12 so it does not exceed minor version number of AWS Aurora compatibility. 

>  All 2.x Aurora MySQL engine versions are wire-compatible with Community MySQL 5.7.12. 
https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraMySQL.Updates.Versions.html

Reply:
Addressed https://github.com/dvsa/cvs-tsk-update-store/pull/1/commits/6a6ed90b3eb3a0025891607808dde506ac7aec0d